src/main/java/io/cdap/plugin/google/sheets/source/GoogleSheetsSourceConfig.java

```diff
@@ -186,12 +186,12 @@ public class GoogleSheetsSourceConfig extends GoogleFilteringSourceConfig {
 
   @Name(LAST_DATA_COLUMN)
   @Description("Last column plugin will read as data. It will be ignored if the Column Names " +
-    "Row contain less number of columns.")
+    "Row contain less number of columns. Set it to 0 to read all the columns in the sheet.")
```

Comment:
was this approach approved? We had discussed in the design doc to not go ahead with it instead always use auto-detect and hide these fields?